src/collector.ts

```diff
@@ -128,7 +128,7 @@ export const collect = async (inUrl: string, args: CollectorOptions) => {
             };
         }
         logger.info(`Started Puppeteer with pid ${browser.process().pid}`);
-        page = (await browser.pages())[0];
+        page = await browser.newPage();
```

Comment:
I think the reason we did it this way is to avoid having a useless `about:blank`. It's purely aesthetic I realize, but this does feel slightly worse...
<img width="621" alt="Screenshot 2024-08-01 at 5 29 17 PM" src="https://github.com/user-attachments/assets/0657a944-57cd-4832-b6ff-4fb164b07797">

Reply:
ahhhh interesting. I'll revert it then. I thought for a minute that the syntax change was fixing the sacbee issue, but it didn't.

Reply:
yeah, you never know, if it did fix sacbee I would be all in!